src/ripple/ripple.js

```diff
@@ -118,7 +119,12 @@ MaterialRipple.prototype.upHandler_ = function(event) {
 
   // Don't fire for the artificial "mouseup" generated by a double-click.
   if (event && event.detail !== 2) {
-    this.rippleElement_.classList.remove(this.CssClasses_.IS_VISIBLE);
+    // Allow a repaint to occur before removing this class, so the animation
+    // shows for tap events, which seem to trigger a mouseup too soon after
+    // mousedown.
+    window.setTimeout(function() {
```

Comment:
> In this particular case we aren't looking at frames

Ah, from the PR title I assumed we were actually looking for a proper repaint, which is why I thought `requestAnimationFrame` would be more appropriate. If we are not looking for a repaint, `setTimeout(...,0)` should do fine (although I still like the semantic value of rAF.

@cainmi Could you test which of these two approaches actually work?